CMakeLists.txt

```diff
@@ -744,6 +744,14 @@ if (Gettext_FOUND)
     list(APPEND devilutionx_lang_targets "${_lang_target}")
     list(APPEND devilutionx_lang_files "${_gmo_file}")
 
+    if(APPLE)
+      set_source_files_properties("${_gmo_file}" PROPERTIES
+        MACOSX_PACKAGE_LOCATION "Resources/assets/${lang}.gmo"
+        XCODE_EXPLICIT_FILE_TYPE compiled)
+      add_dependencies(${BIN_TARGET} "${_gmo_file}" "${_lang_target}")
+      target_sources(${BIN_TARGET} PRIVATE "${_gmo_file}")

```

Comment:
```

CMake Error at CMakeLists.txt:751 (add_dependencies):
  The dependency target
  "/Users/runner/work/devilutionX/devilutionX/build/assets/bg.gmo" of target
  "devilutionx" does not exist.
```

Reply:
Can you please try again?

Reply:
I've just force-pushed a fix to `MACOSX_PACKAGE_LOCATION` (it's a directory, not a filename)